jekyll-run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -euo pipefail
+
+docker run -it --rm -v ./static:/srv/jekyll jekyll /bin/bash -c "bundle exec jekyll build -w"
```

Comment:
Also, bash's signal forwarding is surprising enough that I'd probably avoid using it here (it's probably just a process supervisor mostly?).

Instead, maybe try some variation on:

```
docker run -it --rm --init ... bundle exec jekyll build -w
```

with no `/bin/bash`: the `--init` flag gives you a pid 1 that does a much more better job (i.e. it reacts to `docker kill` as you'd expect, etc.).